tests/ci_build/install/ubuntu_install_python_package.sh

```diff
@@ -1,3 +1,3 @@
 # install libraries for python package on ubuntu
-pip2 install nose pylint numpy nose-timer cython decorator scipy tornado
-pip3 install nose pylint numpy nose-timer cython decorator scipy tornado typed_ast
+pip2 install nose pylint numpy nose-timer cython decorator scipy xgboost psutil tornado
```

Comment:
move xgboost psutil, and redis to ubuntu_install_redis.sh, just to make transition easier.